force-app/main/default/lwc/wireListView/__tests__/wireListView.test.js

```diff
@@ -0,0 +1,54 @@
+import { createElement } from 'lwc';
+import WireListView from 'c/wireListView';
+import { getListUi } from 'lightning/uiListApi';
+import { registerLdsTestWireAdapter } from '@salesforce/lwc-jest';
+
+const mockGetListUi = require('./data/getListUi.json');
+
+// Register as an LDS wire adapter. Some tests verify the provisioned values trigger desired behavior.
+const getListUiAdapter = registerLdsTestWireAdapter(getListUi);
+
+describe('c-wire-list-view', () => {
+    afterEach(() => {
+        // The jsdom instance is shared across test cases in a single file so reset the DOM
+        while (document.body.firstChild) {
+            document.body.removeChild(document.body.firstChild);
+        }
+    });
+
+    describe('getListUi @wire data', () => {
+        it('renders contacts from listView', () => {
+            const CONTACT_RESULT = 'Amy Taylor';
+
+            // Create element
+            const element = createElement('c-wire-list-view', {
+                is: WireListView
+            });
+            document.body.appendChild(element);
+
+            getListUiAdapter.emit(mockGetListUi);
+
+            return Promise.resolve().then(() => {
+                const contactEls = element.shadowRoot.querySelectorAll('p');
+                expect(contactEls.length).toBe(6);
```

Comment:
Prefer to have this number and the contact name above extracted from imported test data.